pkg/controller/registry/resolver/resolver.go

```diff
@@ -114,10 +123,6 @@ func (r *Resolver) Resolve(namespaces []string, subs []*v1alpha1.Subscription) (
 
 	r.addInvariants(namespacedCache, variables)
 
-	if err := namespacedCache.Error(); err != nil {
```

Comment:
There are cache sources that do not correspond to registry pods (like the CSVs that are already installed), so removing this ignores more errors than you're intending.